traffic-advice.md

```diff
@@ -0,0 +1,32 @@
+# Traffic Advice
+
+Publishers may wish not to accept traffic from [private prefetch proxies](README.md) and other sources other than direct user traffic, for instance to reduce server load due to speculative prefetch activity.
+
+We propose a well-known "traffic advice" resource, analogous to `/robots.txt` (for web crawlers), which allows an HTTP server to declare that implementing agents should stop sending traffic to it for some time.
+
+## Proposal
+
+Agents which respect traffic advice should fetch the well-known path `/.well-known/traffic-advice.json`. If it returns a `200 OK` response with the `application/json` MIME type, the response body should contain valid JSON like the following:
```

Comment:
Yeah, right now it's not clear whether the web browser would consult this file before using a proxy, versus the proxy doing this without the web browser in the loop.

Reply:
I've added an example at the bottom. For a prefetch proxy, it makes the most sense for the proxy to do this (as it can easily and transparently cache the conclusion).